### PR TITLE
[zh] Localize distributed_app & distributed_systems

### PR DESCRIPTION
**Pull request by Jacob953**
**Done**
- [x] Add distributed_app.md to dev-zh.
- [x] Add distributed_systems.md to dev-zh.

### CODE REVIEW

File: content/zh/distributed_systems.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+如果没有分布式计算，我们今天依赖的许多应用就不可能实现。
+
+传统上，系统会纵向 [扩展](/scalability/)。这就是当你在一台单独的机器上添加更多的 CPU 或内存。
+垂直扩展很耗时，需要停机，而且很快就会达到极限。
+
+## 如何帮助
+
+分布式系统允许 [水平扩展](/horizontal_scaling/)（例如，在需要时向系统添加更多节点）。这可以是自动化的，允许系统处理工作负荷或资源消耗的突然增加。
+
+非分布式系统面临着失败的风险，因为如果一台机器发生故障，整个系统都会失败。分布式系统可以被设计成这样的方式，即使一些机器发生故障，整个系统仍然可以继续工作，产生相同的结果。
```
Comment (Rocksnake) [inline content/zh/distributed_systems.md:26]:
```suggestion
非分布式系统面临着故障的风险，因为如果一台机器发生故障，整个系统都会故障。分布式系统可以设计成这样，即使一些机器发生故障，整个系统仍然可以继续工作，产生相同的结果。
```

**Comment by Jacob953** [comment]
@Rocksnake Thx! I'd fixed it. :)

**Review by Rocksnake — APPROVED** [review]
Thanks！LGTM

**Review by hanyuancheung — APPROVED** [review]
lgtm

**Pull request merged**